zynq_boot_bin/build_boot_bin.ps1

```diff
@@ -28,9 +27,35 @@ if (!(Test-Path $XSA_FILE)) {
     usage
 }
 
-if (!(Test-Path $UBOOT_FILE)) {
-    echo "$UBOOT_FILE not found"
-    usage
+$patterns = @("zed", "ccfmc_.*", "ccbob_.*", "usrpe31x", "zc702", "zc706", "coraz7s")
+$regex = $patterns -join '|'
+$fullPath = (Get-Item $XSA_FILE).FullName
+$dir = Split-Path $fullPath
+$newName = (Get-Item $XSA_FILE).BaseName + ".zip"
+$renamedFilePath = Join-Path $dir $newName
+$extractLocation = Join-Path $dir "extractedXSA"
+Copy-Item $XSA_FILE $renamedFilePath
+Expand-Archive -Path $renamedFilePath -DestinationPath $extractLocation -Force
+
+# Search inside extracted files
+$line = Get-ChildItem -Path $extractLocation\*.hwh -Recurse -File |
+    Get-Content |
+    Select-String -Pattern "PATH_TO_FILE" |
+	Select-Object -First 1 -ExpandProperty Line
+$carrier = [regex]::Match($line, $regex).Value	
+
+switch -Wildcard ($carrier) {
```

Comment:
Add a comment there that we offer prebuilt u-boot... but they come also with the own one.